models/issues/issue_project.go

```diff
@@ -28,6 +28,25 @@ func (issue *Issue) LoadProject(ctx context.Context) (err error) {
 	return err
 }
 
+func (issue *Issue) LoadProjectIssue(ctx context.Context) (err error) {
```

Comment:
Why not use `LoadProject` directly?

Reply:
this used to load project column, not for project.